pkg/kv/kvclient/kvcoord/dist_sender_rangefeed.go

```diff
@@ -520,6 +515,7 @@ func newActiveRangeFeed(
 			Span:        span,
 			StartAfter:  startAfter,
 			CreatedTime: timeutil.Now(),
+			InCatchup:   true,
```

Comment:
I don't think we should set this anymore?

Reply:
ack.  Thanks.